### PR TITLE
Comments

### PR DESCRIPTION
**Pull request by eileenmcnaughton**

### CODE REVIEW

File: CRM/ACL/Form/WordPress/Permissions.php [hunk 1]
```diff
@@ -134,7 +134,7 @@ public function postProcess() {
         // Get the permissions into a format that matches what we get from WP
         $allWarningPermissions = CRM_Core_Permission::getAnonymousPermissionsWarnings();
         foreach ($allWarningPermissions as $key => $permission) {
-          $allWarningPermissions[$key] = CRM_utils_String::munge(strtolower($permission));
+          $allWarningPermissions[$key] = CRM_Utils_String::munge(strtolower($permission));
```
Comment (eileenmcnaughton) [inline CRM/ACL/Form/WordPress/Permissions.php]: fix casing to be correct

File: CRM/Case/Form/Activity.php [hunk 2]
```diff
@@ -341,7 +341,7 @@ public static function formRule($fields, $files, $self) {
       return TRUE;
     }
 
-    return parent::formrule($fields, $files, $self);
+    return parent::formRule($fields, $files, $self);
```
Comment (eileenmcnaughton) [inline CRM/Case/Form/Activity.php]: fix casing to be correct

File: CRM/Case/Form/Task/Print.php [hunk 3]
```diff
@@ -41,7 +41,7 @@ class CRM_Case_Form_Task_Print extends CRM_Case_Form_Task {
    * Build all the data structures needed to build the form.
    */
   public function preProcess() {
-    parent::preprocess();
+    parent::preProcess();
```
Comment (eileenmcnaughton) [inline CRM/Case/Form/Task/Print.php]: fix casing to be correct

File: CRM/Core/DAO/AllCoreTables.php [hunk 4]
```diff
@@ -206,11 +235,12 @@ public static function getExports($dao, $labelName, $prefix, $foreignDAOs) {
       $exports = array();
       $fields = $dao::fields();
 
-      foreach($fields as $name => $field) {
+      foreach ($fields as $name => $field) {
```
Comment (eileenmcnaughton) [inline CRM/Core/DAO/AllCoreTables.php]: whitespace - this file is probably ignored as it used to be autogenerated

File: CRM/Core/Payment/AuthorizeNet.php [hunk 5]
```diff
@@ -816,7 +816,7 @@ public function changeSubscriptionAmount(&$message = '', $params = array()) {
 
     curl_close($submit);
 
-    $responseFields = $this->_ParseArbReturn($response);
+    $responseFields = $this->_parseArbReturn($response);
```
Comment (eileenmcnaughton) [inline CRM/Core/Payment/AuthorizeNet.php]: fix casing to be correct

File: xml/templates/civicrm_version.tpl [hunk 6]
```diff
@@ -1,10 +1,12 @@
 <?php
-
+/**
```
Comment (eileenmcnaughton) [inline xml/templates/civicrm_version.tpl]: template for civicrm.version.php

File: xml/templates/schema_structure.tpl [hunk 7]
```diff
@@ -36,7 +36,14 @@
  */
 class CRM_Core_I18n_SchemaStructure
 {ldelim}
-    static function &columns()
```
Comment (eileenmcnaughton) [inline xml/templates/schema_structure.tpl]: template for SchemaStructure

File: xml/templates/schema_structure.tpl [hunk 8]
```diff
@@ -36,7 +36,14 @@
...
+
+    /**
+     * Get translatable columns.
+     *
+     * @return array
+     *   A table-indexed array of translatable columns.
+     */
+    public static function &columns()
```
Comment (eileenmcnaughton) [inline xml/templates/schema_structure.tpl]: I desperately want to remove the & too but not mixing it in

File: CRM/Mailing/BAO/Mailing.php [hunk 9]
```diff
@@ -3173,10 +3173,11 @@ public static function getWorkflowFieldPerms() {
   }
 
   /**
-   * Whitelist of possible values for the entity_table field
+   * White-list of possible values for the entity_table field.
+   *
    * @return array
    */
-  public static function mailingGroupEntityTables($context = NULL) {
```
Comment (seamuslee001) [inline CRM/Mailing/BAO/Mailing.php]: I would check with @totten on this one

File: CRM/PCP/Form/Campaign.php [hunk 10]
```diff
@@ -64,8 +64,14 @@ public function preProcess() {
...
   }
 
+  /**
+   * Set default form values.
+   *
+   * @return array
+   *   Default values for the form.
+   */
   public function setDefaultValues() {
-    $dafaults = array();
```
Comment (seamuslee001) [inline CRM/PCP/Form/Campaign.php]: ha

**Review by seamuslee001 — APPROVED** [review]
Looks good apart from the one in the mailing

File: CRM/Mailing/BAO/Mailing.php (hunk 9 above)
Comment (eileenmcnaughton) [inline CRM/Mailing/BAO/Mailing.php]: Actually it looks like it was @colemanw who edited that fn so the var is not removed without removing it - any reason C?

Comment (colemanw) [inline CRM/Mailing/BAO/Mailing.php]: Ah, I see. The function works kind of like a hook, in that the caller will pass in that variable whenever calling a `*EntityTables` function whether it's needed or not. In this case it's not needed. Fine to remove it or document it.

File: CRM/PCP/Form/Campaign.php (hunk 10 above)
Comment (colemanw) [inline CRM/PCP/Form/Campaign.php]: Funny thing is that php won't care. Unless of course the two conditionals below are false. I guess one or the other is always true or else people would have noticed an undefined var notice.

**Pull request merged**